### DISCUSSION TITLE
Toaster

### DISCUSSION THREAD

**Discussion by lfbergee** [fremtind/jokul#1718] [Annet]
Forslag til nytt mønster, i noen tilfeller er det nyttig å gi brukeren tilbakemelding at noe har skjedd eller at et system har respondert, uten at brukeren skal sendes et nytt sted, eller at man trenger en "permanent" tilbakemelding ala MessageBox eller AlertMessage.

Feks om man har et system hvor brukeren kan mellomlagre innholdet i et tekstfelt på serveren, da kan det være nyttig å gi en "Svar lagret", som kan midlertidig poppe inn å betrygge brukeren at forventet operasjon har skjedd.

Et mulig pattern for dette er en Toaster eller Growl-style notifikasjon. En liten boks som dukker opp, gir beskjeden sin og forsvinner etter noen sekunder.

https://user-images.githubusercontent.com/46530368/103543361-ae3d8a00-4e9e-11eb-8196-ad32687e0188.mp4

**Comment by Slaine2** [comment]
Kanskje dette kan løses litt mer subtilt? En liten animasjon kan også hjelpe øyet til å fokusere på den uten at brødristeren må være så stor.

**Reply by lfbergee** [reply]
Jepp, ingen designere som har sett på denne enda, så om du har lyst å komme med et forslag, så kan jeg oppdatere :)

**Reply by Slaine2** [reply]
Wicked, så kan jeg gjøre litt undersøkelser 🔢  kom tilbake til det med en eksempel
